memberships/models.py

```diff
@@ -16,6 +20,23 @@ class Member(models.Model):
     constitution_agreed = models.BooleanField()
     stripe_customer_id = models.CharField(max_length=255)
     user = models.OneToOneField(User, on_delete=models.CASCADE)
+    profile_image = models.ImageField(upload_to="images/", blank=True)
+    telephone = models.CharField(max_length=255, blank=True)
+    minecraft_username = models.CharField(max_length=255, blank=True)
+    address_1 = models.CharField(max_length=255, blank=True)
+    address_postcode = models.CharField(max_length=10, blank=True)
+    gift_aid = models.BooleanField(default=False)
+    gdpr_likeness = models.BooleanField(default=False)
+    gdpr_sms_updates = models.BooleanField(default=False)
+    gdpr_sms_notifications = models.BooleanField(default=False)
+    gdpr_email_updates = models.BooleanField(default=False)
+    gdpr_email_notifications = models.BooleanField(default=False)
+    gdpr_telephone_updates = models.BooleanField(default=False)
+    gdpr_telephone_notifications = models.BooleanField(default=False)
+    gdpr_post_updates = models.BooleanField(default=False)
+    gdpr_post_notifications = models.BooleanField(default=False)
+    # renewal_date = models.DateField()
```

Comment:
```suggestion

```

Reply:
I have uncommented this, as we will need a way to tell the user when they need to renew. Not sure why it was commented out.